packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.tsx

```diff
@@ -52,7 +52,8 @@ export class ProgressIndicator extends BaseComponent<IProgressIndicatorProps, {}
             role='progressbar'
             aria-valuemin='0'
             aria-valuemax='100'
-            aria-valuenow={ percentComplete.toFixed().toString() }>
+            aria-valuenow={ percentComplete.toFixed().toString() }
+            aria-valuetext={ ariaValueText }>
```

Comment:
if ariaValueText is undefined, is having this here bad? Does this need to be conditional?